packages/config/src/config.test.js

```diff
@@ -153,4 +162,176 @@ describe("config()", () => {
+          })
+        })
+
+        describe("with an array of configs loaded", () => {
+          beforeEach(async () => {
+            /**
+             * @type {FlowJSONv1}
+             */
+            let secondFlowJSON = {
+              accounts: {},
+              contracts: {
+                ThirdContract: {
+                  aliases: {
+                    emulator: "0x3",
+                    testnet: "0x03",
+                    mainnet: "0x003",
+                  },
+                },
+              },
+              deployments: {},
+              networks: {},
+            }
+
+            await config().load({flowJSON: [flowJSON, secondFlowJSON]})
+            await idle()
+          })
+
+          test("should load the contract locations from the secondFlowJSON", async () => {
+            await expect(config().get("0xThirdContract")).resolves.toBe("0x3")
+          })
+        })
+      })
+
+      describe("flow.json v2", () => {
+        /**
+         * @type {FlowJSONv2}
+         */
+        let flowJSON
+
+        describe("contracts with an account alias", () => {
+          beforeEach(async () => {
+            flowJSON = {
+              contracts: {
+                HelloWorld: "alice",
+              },
+              networks: {
+                emulator: "127.0.0.1:3569",
+              },
+              accounts: {
+                alice: {
+                  emulator: "0x1",
+                  testnet: "0x01",
+                },
+                bob: {
+                  emulator: "0x2",
+                  testnet: "0x02",
+                },
+              },
+            }
+
+            await config().load({flowJSON})
+            await idle()
+          })
+
+          test("should load the contract location wrt the currently set network", async () => {
+            await expect(config().get("0xHelloWorld")).resolves.toBe("0x1")
+          })
+        })
+
+        describe("contract with multiple sources", () => {
+          beforeEach(async () => {
+            flowJSON = {
+              contracts: {
+                MultiSourceContract: {
+                  emulator: "./contracts/MultiSourceContract.cdc",
+                  testnet: "bob",
+                },
+              },
+              networks: {
+                emulator: "127.0.0.1:3569",
+              },
+              accounts: {
+                alice: {
+                  emulator: "0x1",
+                  testnet: "0x01",
+                },
+                bob: {
+                  emulator: "0x2",
+                  testnet: "0x02",
+                },
+              },
+            }
+
+            await config().load({flowJSON})
+            await idle()
+          })
+
+          test("should load the contract location wrt the currently set network", async () => {
+            await expect(config().get("0xMultiSourceContract")).resolves.toBe(
+              "./contracts/MultiSourceContract.cdc"
```

Comment:
In the v2 case I think this actually resolves to an address

Reply:
In the case of the emulator it can be.